### DISCUSSION TITLE
协同编辑图片如何保存和读取

### DISCUSSION THREAD

**Discussion by xuxin-git** [dream-num/Luckysheet#750] [Q&A]
协同编辑时，配了自定义同步图片customImageUpdateMethodConfig，![image](https://user-images.githubusercontent.com/68700557/129168995-fe652dfd-58b4-4371-8ad5-df6702f57e50.png)。如何回显，以及同步到其他客户端

**Comment by javahuangjia** [comment]
张三李四

**Comment by 858882847** [comment]
> 协同编辑时，配了自定义同步图片customImageUpdateMethodConfig，![image](https://user-images.githubusercontent.com/68700557/129168995-fe652dfd-58b4-4371-8ad5-df6702f57e50.png)。如何回显，以及同步到其他客户端

Have you ever encountered a situation where the imported image has a location, but the location of the saved image remains unchanged after changing the rows and columns
